packages/block-library/src/image/edit.js

```diff
@@ -90,8 +92,9 @@ export function ImageEdit( {
 		width,
 		height,
 		sizeSlug,
+		inhertedAttributes,
```

Comment:
Typo
```suggestion
		inheritedAttributes,
```

Edit: I think that it could be removed since we use `inheritedAttributes` from `attributes` in the `Image` component. https://github.com/WordPress/gutenberg/pull/25940/files#diff-a14b61c874095649b236b78413e805b4541277b24eb71ce350a17d9f2b6396b1R75

Reply:
thanks, have removed.